pom.xml

```diff
@@ -85,7 +86,8 @@
     <scm>
         <url>http://github.com/dbs-leipzig/gradoop.git</url>
         <connection>scm:git:git@github.com:dbs-leipzig/gradoop.git</connection>
-        <developerConnection>scm:git:dbs-leipzig@github.com/dbs-leipzig/gradoop.git</developerConnection>
+        <developerConnection>scm:git:dbs-leipzig@github.com/dbs-leipzig/gradoop.git
```

Comment:
If you make a line break, move content to a new line between opening and closing tag